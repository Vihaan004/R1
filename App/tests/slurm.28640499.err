
=====================================================================
This module is only for creating or activating Python environments:
$ mamba create -n myenv -c conda-forge python=3 <package_name>
$ source activate myenv

Only run "pip install" after activating an environment.
Running pip without activating an environment is known to cause issues.

To list available environments:
$ mamba info --envs

Other uses are not tested. More info: https://links.asu.edu/solpy
=====================================================================

  
Lmod has detected the following error: The following module(s) are unknown:
"cuda/12.0"

Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore_cache load "cuda/12.0"

Also make sure that all modulefiles written in TCL start with the string
#%Module



/home/vpatel69/.local/lib/python3.12/site-packages/cupy/_environment.py:541: UserWarning: 
--------------------------------------------------------------------------------

  CuPy may not function correctly because multiple CuPy packages are installed
  in your environment:

    cupy, cupy-cuda11x

  Follow these steps to resolve this issue:

    1. For all packages listed above, run the following command to remove all
       existing CuPy installations:

         $ pip uninstall <package_name>

      If you previously installed CuPy via conda, also run the following:

         $ conda uninstall cupy

    2. Install the appropriate CuPy package.
       Refer to the Installation Guide for detailed instructions.

         https://docs.cupy.dev/en/stable/install.html

--------------------------------------------------------------------------------

  warnings.warn(f'''
Traceback (most recent call last):
  File "/home/vpatel69/R1/App/tests/matrix_operations_gpu.py", line 173, in <module>
    main()
  File "/home/vpatel69/R1/App/tests/matrix_operations_gpu.py", line 18, in main
    print(f"GPU Device: {cp.cuda.get_device_name()}")
                         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/vpatel69/.local/lib/python3.12/site-packages/cupy/cuda/__init__.py", line 84, in __getattr__
    raise AttributeError(
AttributeError: module 'cupy.cuda' has no attribute 'get_device_name'. Did you mean: 'get_device_id'?
