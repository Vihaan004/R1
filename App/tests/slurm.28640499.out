==================================================
GPU Matrix Operations Test
Using CuPy (RAPIDS 25.02) for GPU computation
==================================================
